src/Nancy.Testing.Tests/ConfigurableBootstrapperFixture.cs

```diff
@@ -219,5 +219,21 @@ public BlowUpModule()
                 Get["/"] = _ => { throw new InvalidOperationException("Oh noes!"); };
             }
         }
+
+        private static class Record

```

Comment:
I think it does? `Assert.Throws<T>` and `Assert.ThrowsAsync<T>` both first assert the correct exception and then returns it.